integration/env/common.go

```diff
@@ -137,10 +140,19 @@ func ClusterID() string {
 
 	parts = append(parts, "ci")
 	parts = append(parts, TestedVersion()[0:3])
-	parts = append(parts, CircleSHA()[0:5])
-	if TestHash() != "" {
-		parts = append(parts, TestHash())
-	}
+	shaPart := CircleSHA()[0:4]
+	testPart := TestHash()
```

Comment:
AFAIK TestHash is hash of integration test directory, so we have some unique identificator across parallel tests.

The magic is there because i need to somehow reduce the number of letters in clusterID so I do hash from the 2 hashes.